lib/mayaHydra/hydraExtensions/plugInfo.json

```diff
@@ -85,6 +85,12 @@
                         ],
                         "displayName": "Ai SkyDome Light in Hydra for Maya."
                     },
+					"MayaHydraAiAreaLightAdapter": {
```

Comment:
remove the extra space?

Reply:
I have tried right now, but there is no extra space it seems.

Reply:
![image](https://github.com/user-attachments/assets/0b0cbe24-1b4c-41f5-9d2b-5456cbe820b0)

Reply:
I think it's tab, you can enable option in VS "Eidt"->"Advanced"->"View White Space"

![tab](https://github.com/user-attachments/assets/381694dd-158e-4ec0-aeea-a32f72017207)

Reply:
Seems ok now. Thanks.

Reply:
BTW, I ususally force "Tab" as "Space" in VS when you're using VS to Edit file, as "Tab" is system dependent.
![set-tab-vs](https://github.com/user-attachments/assets/28c22a1c-b07a-483e-bb02-eb6ec3e240d4)

Reply:
Should be good now , thanks